dipy/reconst/utils.py

```diff
@@ -49,6 +49,74 @@ def dki_design_matrix(gtab):
 
     return B
 
+def cti_design_matrix(gtab1, gtab2): 
+	r"""Construct B design matrix for CTI.
+	
+	Parameters
+	----------
+	gtab1: dipy.core.gradients.GradientTable
+		A GradientTable class instance
```

Comment:
Specific that this is "A GradientTable class instance for first DDE diffusion epoch"